Philips.CodeAnalysis.Test/Helpers/CodeFixVerifier.Helper.cs

```diff
@@ -79,7 +79,7 @@ private static string GetStringFromDocument(Document document)
 			SyntaxNode syntaxNode = document.GetSyntaxRootAsync().Result;
 			if (syntaxNode != null)
 			{
-				var vv = syntaxNode.GetText().ToString();
+				_ = syntaxNode.GetText().ToString();
```

Comment:
Seems like a bug to me that this result is not used

Reply:
I'm guessing it was there for debugging purposes only. Removed the code. Add Issue to investigate automation.